sdk/eventhub/azure-eventhub/CLIENT_DEVELOPER.md

```diff
@@ -33,6 +33,12 @@ This guide is intended for developers contributing to the Azure Event Hubs Pytho
    pip install -e .
    ```
 
+   **Note**: If the editable install fails or you encounter import errors, you may need to remove checkpoint store packages first:
+   ```bash
+   pip uninstall azure-eventhub-checkpointstoreblob azure-eventhub-checkpointstoreblob-aio -y
+   pip install -e .
+   ```
```

Comment:
Note, or alternatively, you can install the checkpointstoreblob packages, but will need to pip install the client library in non-editable mode.